### PR TITLE
[CARE-1613] Exclude private stories in search engines

### PR DESCRIPTION
**Pull request by MohammadxAli**

### CODE REVIEW

File: packages/nextjs/src/components-nextjs/StorySeo/StorySeo.tsx [hunk 1]
```diff
@@ -43,8 +43,8 @@ export function StorySeo({ story, noindex }: Props) {
                 title={seoTitle}
                 description={seoDescription}
                 canonical={canonical}
-                noindex={noindex ?? !indexable}
-                nofollow={noindex ?? !indexable}
+                noindex={noindex && !indexable}
+                nofollow={noindex && !indexable}
```
Comment (e1himself) [inline packages/nextjs/src/components-nextjs/StorySeo/StorySeo.tsx:47]: After thinking more, I believe it should be like this:

```tsx
noindex={noindex || !indexable}
nofollow={noindex || !indexable}
```

Same change for `PageSeo.tsx`

**Review by e1himself — CHANGES_REQUESTED** [review]
See my comment above

File: packages/nextjs/src/components-nextjs/StorySeo/StorySeo.tsx (hunk 1 above)
Comment (oleg-semyonov) [reply packages/nextjs/src/components-nextjs/StorySeo/StorySeo.tsx]: Was about to post the same comment 😃 
With current code, the seo setting will not take effect if the site is indexable

**Review by oleg-semyonov — APPROVED** [review]

File: packages/nextjs/src/components-nextjs/StorySeo/StorySeo.tsx (hunk 1 above)
Comment (e1himself) [reply packages/nextjs/src/components-nextjs/StorySeo/StorySeo.tsx]: Inverted boolean flags are tricky! :)

**Pull request merged**